package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "private": true,
+  "scripts": {
+    "pretest": "dotnet fable Fuchu.FableTests --outDir bin",
+    "test": "node --eval \"require('esm')(module)('./bin/Main.js')\" ./bin/Main.js ./bin/Main.js ./bin /d"
```

Comment:
Sorry @wallymathieu, forgot to tell there's an option in Fable `--runScript` that basically does this. I think something like this should work:

```
"test": "dotnet fable Fuchu.FableTests --outDir bin --runScript ./bin /d"
```